dataland-frontend/tests/component/components/pages/CompanyCockpitPage.cy.ts

```diff
@@ -248,8 +281,8 @@ describe("Component test for the company cockpit", () => {
       validateBackButtonExistence(true);
       validateSearchBarExistence(false);
       validateCompanyInformationBanner();
-      validateClaimOwnershipPanel(true);
-      validateFrameworkSummaryPanels(false);
+      validateClaimOwnershipPanel(isClaimOwnershipPanelExpected);
+      validateFrameworkSummaryPanels(isProvideDataButtonExpected);
```

Comment:
In Zeile 261 wird auch validateSingleDataRequestButton verwendet. Kannst du da vll auch den Boolean als Variable setzen, wie das in Zeile 253 ist?

Reply:
Danke

Reply:
Ja klar. Habe jetzt überall, wo kein validateAbcDefExistence() den boolean gelassen. Das ist in meinen Augen verständlich. Bei den validateAbcDef() habe ich den boolean als Variable genutzt. Damit sollten die Tests verständlicher sein.

Reply:
Sieht gut aus